### DISCUSSION TITLE
You can suggest optimizations to our Weaver

### DISCUSSION THREAD

**Discussion by yolandadadada** [IBAX-io/go-ibax#1927] [Ideas]
You can suggest optimizations to our Weaver!
1. Any place you feel inconvenient.
2. Features that can be added in the future.

**Comment by HeybetX** [comment]
Arayüzü çok kötü

**Reply by akmol1076** [reply]
Wonderful a great strong project

**Reply by Oguz1981** [reply]
Uygulama çok kötü

**Reply by cerkescnblt** [reply]
başarılı olacagınızı sanmıyorum çok uzun aürdü hala bişey yok

**Reply by christ360** [reply]
Cool

**Reply by Mdnurunnabi1** [reply]
Good project management

**Reply by verdrop2021** [reply]
good project

**Reply by Ireenfeni** [reply]
Good

**Reply by Sitoci** [reply]
bazen akışına bırakmak gerek

**Reply by Sitoci** [reply]
kervanı bazen de yolda düzmek gerekir

**Reply by ybalkay** [reply]
ne yapıyoruz la biz burada,

**Reply by onalsol** [reply]
Proje gelecek vaadediyor tabi yazildigi gibi olurda testnet baslarsa

**Reply by franssianipar** [reply]
so great

**Reply by cerkescnblt** [reply]
hızlandırmK gerekitor projeyi çok agır ilerliyor

**Reply by painter123** [reply]
Puanım var ibcx yok

**Reply by VoxPardus** [reply]
nice

**Reply by trungcba** [reply]
nice

**Reply by LuuHung24851525** [reply]
good

**Reply by snoopyeth** [reply]
Nice

**Reply by amulla505** [reply]
Unique

**Reply by shahidul40** [reply]
good

**Reply by maxwin96** [reply]
lfg

**Reply by Winnode** [reply]
great

**Reply by OneNov0209** [reply]
Amazing

**Reply by sxlmnwb** [reply]
hii

**Reply by sugiarto96** [reply]
Nice

**Reply by Egi08** [reply]
great

**Reply by Ndod26** [reply]
Nice

**Reply by Ifra999** [reply]
nice

**Reply by rastae** [reply]
good

**Reply by 0xheycat** [reply]
thank you

**Reply by ovi2258** [reply]
nice

**Reply by Kheisya01** [reply]
I hope i can get the reward

**Reply by Tintnaingu** [reply]
Good idea 👍

**Reply by noue125** [reply]
> Arayüzü çok kötü



> You can suggest optimizations to our Weaver!
> 
> 1. Any place you feel inconvenient.
> 2. Features that can be added in the future.

Nothing

**Reply by giwing3** [reply]
You can suggest optimizations to our Weaver!

Any place you feel inconvenient.
Features that can be added in the future.

**Reply by arnyo** [reply]
Good

**Reply by Apojanti** [reply]
O KADAR KÖTÜ Kİ İŞLEM YAPAMIYORUZ  CÜZDAN BÖLÜMÜ DE ÇOK SAÇMA

**Reply by Ugurkayaaa** [reply]
kesinlikle

**Comment by CODOMAN** [comment]
Set up this project as soon as possible Ok 👌 Git hub turkish come now ibax wallet application come, let's set up the wallet there and everything will be clear

**Reply by ozkangngr** [reply]
We are looking forward to the developments of a great project, I feel very good things will happen, thank you to the project team.

**Reply by EduardoALzola** [reply]
I love this IBAX project, it's super

**Reply by franssianipar** [reply]
love

**Reply by trungcba** [reply]
ok

**Reply by Syarif325** [reply]
nice project

**Comment by CODOMAN** [comment]
Be clear, whether it's distribution, coins, wallets, how long to wait, apps to wait for, all right.

**Reply by Kulyt5** [reply]
Good

**Reply by Dodyharyanto** [reply]
good bro

**Reply by rastae** [reply]
nice

**Reply by Pokrek2709** [reply]
> Be clear, whether it's distribution, coins, wallets, how long to wait, apps to wait for, all right.

good

**Comment by iismailtosun** [comment]
Nerede coinler

**Reply by ByTeachers** [reply]
I have no doubt that IBAX is a very powerful project. I also know that the team is working hard. But there are problems with the airdrop. It seems that those who have worked for months and supported this project will not be able to get their money's worth. I think it would be fairer to base the ongoing scores instead of applying a method like before and after February 1st. Of course, the team knows best. I have full confidence in this team. Thank you for everything. I am happy to know this project.

**Reply by bycoin1** [reply]
Proje henüz tamamlanmadı hocam ama büyük olacak emin olabilirsin

**Comment by yunusm87** [comment]
Great Project

**Comment by doganayby** [comment]
it seems like the studies are progressing slowly

**Comment by serhattrkmngl** [comment]
In terms of use, the wallet creates some problems because it is complex. The user interface should be made more convenient.

**Comment by doganayby** [comment]
people have confusion

**Comment by kartal1617** [comment]
the weaver interface is very colorless. Also, the fonts are very bad. These features need to be fixed first.

**Comment by Serdarkck** [comment]
Anlaşılmayan çok konu var daha da kolaylaştırılmalı

**Comment by CODOMAN** [comment]
I have a tuition for you to do the mobile application layout as soon as possible and get the layout well!! And now put this in order.

**Comment by sgsalman** [comment]
interface must be easy to use

**Comment by shnkrcy** [comment]
The theme could be more beautiful and stylish

**Comment by Fermandar** [comment]
ara yüz daha stabil hale getirilebilir

**Comment by maxisero** [comment]
The project is very successful, but the interface is more classic, if we can make it a little more modern, it will be great

**Reply by Edam2134** [reply]
Set up this project as soon as possible... very good

**Reply by muratta35** [reply]
katılıyorum

**Comment by bhnk** [comment]
it is not clear what you are doing, please be more transparent and clear

**Comment by ozkangngr** [comment]
Sabırsızlıkla yeni gelişmeler beklemekteyiz.

**Comment by mokoko06** [comment]
interface could be more convenient

**Comment by kac4r** [comment]
Daha iyi yerlere

**Comment by hicest11a** [comment]
umarım hak ettiği değeri alır

**Reply by Hkrkmz** [reply]
Umarim

**Comment by laprakon** [comment]
Proje çok başarılı ama  daha klasik biraz daha modern hale getirebilirsek harika olur

**Comment by Yegad** [comment]
Net değil arayüzü iyi değil

**Comment by atifileri** [comment]
chorme tarayıcı eklentili cüzdan güzel olur.
Wallet with chrome browser plugin would be nice.

**Comment by oolukman** [comment]
Interface is very complicating. There is not knowledge about airdrop system. For examle we got points for airdrop from twitter retweet missions and collect some points for airdrop or made referance points for airdrop. Than airdop is closed. 
Now still we are collecting points from open boxes. What is Reputation points for? Will it increase our airdrop coins? Why are we open boxes every day? Team must be more clear

**Comment by omerozan16** [comment]
The project seems to be progressing slowly.

**Comment by bycoin1** [comment]
Geleceğin avax ı olabilir mi ?

**Comment by gelinarmudu** [comment]
körlemesine yapıyoruz bişeyler ama bakalım

**Reply by bycoin1** [reply]
Zengin olacağız bu projeden sabır

**Comment by bycoin1** [comment]
Avax ilk lansmanda airdrop olarak kaç avax gönderdi sizce ?

**Comment by MrDino3813** [comment]
There is an adage that says great power comes great responsibility.
I believe that the Ibax network is a great project and it should be handled as such.

For instance the security of Ibax should be really secure and decentralized as possible. The twitter login method doesn't make it very decentralized. I suggest integrating with "BrightID".

**Reply by Greatzoba** [reply]
Yes very collect

**Comment by Greatzoba** [comment]
Nice project

**Comment by wakamaru587** [comment]
Huge Project

**Comment by perseus36** [comment]
> Weaver'ımıza optimizasyonlar önerebilirsiniz!
> 
> 1. Rahatsız hissettiğin herhangi bir yer.
> 2. İleride eklenebilecek özellikler.

The interface is very complex. The design should be simpler and clearer. Also, new missions should be added.

**Reply by Andrelowen** [reply]
Love it 👍

**Comment by nayapurnomo** [comment]
> You can suggest optimizations to our Weaver!
> 
> 1. Any place you feel inconvenient.
> 2. Features that can be added in the future.

The best team

**Comment by nayapurnomo** [comment]
I like

**Comment by kasbizean** [comment]
Best project ..keep stay

**Comment by Apojanti** [comment]
> Weaver'ımıza optimizasyonlar önerebilirsiniz!
> 
> 1. Rahatsız hissettiğin herhangi bir yer.
> 2. İleride eklenebilecek özellikler.

Merhabalar airdrop puanlamasında ve görevlerinde sıkıntı var görev yapınca sayfanın tamamı yapılmış gözüküyor . puan  gelmiyor .ve ayrıca ibaxın arayüz cüzdanına giremiyorum . şifremi unuttum bölümü yok

**Comment by inhayabusha** [comment]
Great 👍

**Comment by Rdhlzrdy** [comment]
> Anda dapat menyarankan pengoptimalan ke Weaver kami!
> 
> 1. Di mana pun Anda merasa tidak nyaman.
> 2. Fitur yang dapat ditambahkan di masa mendatang.

Goodnight

**Comment by Rdhlzrdy** [comment]
Ok

**Comment by Alameen1234** [comment]
Please u should create a mobile application both for android and iPhone

**Comment by laprakon** [comment]
IBAX'ın çok güçlü bir proje olduğundan hiç şüphem yok. Takımın çok çalıştığını da biliyorum. Ancak airdrop ile ilgili sorunlar var. Aylarca çalışıp bu projeye destek olanlar parasının karşılığını alamayacak gibi görünüyor. 1 Şubat öncesi ve sonrası gibi bir yöntem uygulamak yerine devam eden puanları baz almak daha doğru olur diye düşünüyorum. Tabii ki, ekip en iyisini bilir. Bu takıma güvenim tam. Herşey için teşekkürler. Bu projeyi bildiğim için mutluyum.

**Reply by Lockelamora8** [reply]
Aynen

**Comment by Dosarii** [comment]
So far so good I've been touring the ecosystem

**Comment by zeland2323** [comment]
Thanks

**Comment by orjinim** [comment]
iyi süper
